src/helpers/dataCleaner.tsx

```diff
@@ -0,0 +1,71 @@
+import { getCoordinates, getWeatherData, getAirQualityData} from './apiCalls';
+
+
+export const getAllData = async (query: string) => {
```

Comment:
This looks SO CLEAN! Thank you so much for sorting through all the data we get back and giving it to us a neat little bow to work with! Great job.